docs/app/database/database-management.md

```diff
@@ -1,11 +1,40 @@
-# Database Migrations
+# Database Management
 
+- [Basic operations](#basic-operations)
+  - [Initialize](#initialize)
+  - [Start](#start)
+  - [Destroy and reinitialize](#destroy-and-reinitialize)
 - [Running migrations](#running-migrations)
 - [Creating new migrations](#creating-new-migrations)
 - [Multi-head situations](#multi-head-situations)
-- [Deployment](#deployment)
-  - [Removing a column](#removing-a-column)
-  - [Removing a table](#removing-a-table)
+
+## Basic operations
+### Initialize
+
+When setting up the environment, `make init` will take care of building the database and running a few basic migrations. Should there be a situation where this process needs to be repeated or ran separately, the following command will accomplish this:
```

Comment:
> Should there be a situation where this process needs to be repeated or ran separately

nitpick: i think there's ambiguity in the phrase "this process". when i read it, it sounds like the "this process" refers to setting up the environment, so it leaves me wondering why not just re-run `make init`. can you make it a bit clearer? i defer to how you want to word it, but one idea is something like

> Run `make init-db` to start up a local Postgres database container in a detached state and run any pending migrations. Running `make init` will also automatically calls `make init-db`.

or something like that 🤷

Reply:
That's a good catch. I've reworded it to point out that it's ran automatically during setup so there is no confusion.